python-requirements.txt

```diff
@@ -0,0 +1,7 @@
+coloredlogs==15.0.1
+pyaml==19.12.0
+pyelftools==0.26
+pytest-timeout==1.3.3
+pytest-xdist==1.30.0
+pytest==5.2.2
+virtualenv==15.1.0
```

Comment:
Same with this, do we want to pin virtualenv to 15.1.0 or would >=15.1 would suffice?